src/styles/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: row;
```

Comment:
The `flex-direction: row;` is not necessary here. By default, the direction of flex items is row.